pkg/acsengine/const.go

```diff
@@ -17,6 +17,8 @@ const (
 	DefaultKubernetesMasterSubnet = "10.240.0.0/16"
 	// DefaultKubernetesClusterSubnet specifies the default subnet for pods.
 	DefaultKubernetesClusterSubnet = "10.244.0.0/16"
+	// DefaultDockerBridgeSubnet specifies the default subnet for the docker bridge network for masters and agents.
+	DefaultDockerBridgeSubnet = "172.17.0.5/16"
```

Comment:
cluster I just made was  "Config": [
                {
                    "Subnet": "172.17.0.0/16",
                    "Gateway": "172.17.0.1"
                }
            ]
when I inspected the bridge so can we do 172.17.0.1/16. then I think this should be good, after a rebase etc.